underscore.js

```diff
@@ -84,18 +84,23 @@
     };
   };
 
+  var builtinIteratee;
```

Comment:
I vote for moving the declaration (like this) or changing the rule to a warning. We should also file a bug against ESLint